### PR TITLE
Delete SAML settings

### PR DESCRIPTION
**Pull request by aviadl**
related to https://github.com/descope/etc/issues/2700
+ test

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -3,4 +3,9 @@
     "python.testing.pytestArgs": [
         "tests"
     ],
+    "workbench.colorCustomizations": {
```
Comment (dorsha) [inline .vscode/settings.json:6]: Revert this

File: descope/auth.py [hunk 2]
```diff
@@ -138,9 +138,10 @@ def do_post(
             )
         return response
 
-    def do_delete(self, uri: str, pswd: str = None) -> requests.Response:
+    def do_delete(self, uri: str, params=None, pswd: str = None) -> requests.Response:
```
Comment (dorsha) [inline descope/auth.py:141]: Let's add auth test as well

File: .vscode/settings.json (hunk 1 above)
Comment (aviadl) [reply .vscode/settings.json]: done

File: descope/auth.py (hunk 2 above)
Comment (aviadl) [reply descope/auth.py:141]: done

**Review by dorsha — APPROVED** [review]

**Pull request merged**